src/player.js

```diff
@@ -2601,6 +2605,26 @@ export default class Player extends FakeEventTarget {
     }
   }
 
+  /**
+   * Sets the video track selected by the user.
+   * @returns {void}
+   * @private
+   */
+  _setDefaultVideoTrack(): void {
+    const sortedVideoTracks = this._getVideoTracks().sort((track1: VideoTrack, track2: VideoTrack) => track2.bandwidth - track1.bandwidth);
```

Comment:
why the sort() is need before the find() ?

Reply:
when there are some tracks with the same label/height we want to keep the higher bandwidth, like we do in the [UI](https://github.com/kaltura/playkit-js-ui/blob/9d8a69af54817300355897dbef9d6db1215a67d7/src/components/settings/settings.js#L311)